core/pom.xml

```diff
@@ -155,6 +151,71 @@ limitations under the License.
 
   <profiles>
     <profile>
+      <id>it</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-failsafe-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>failsafe-integration-test</id>
+                <phase/>
```

Comment:
1) Indeed. I think the proper phase is `<phase>none</phase>` here (see below).

2) The idea is to prevent `failsafe-integration-test` execution (that is inherited from the parent pom) for "core" module.
Core module needs special treatment since it executes multiple tests for each database engine (see executions like `failsafe-test-mysql` below).

I guess it would not hurt much if we keep the execution enabled (that is if we remove `<execution><id>failsafe-integration-test</id>` here), however this execution is not expected to perform any useful work, so I disable it.

I can add a comment here to clarify why the particular execution is disabled here. Does it make sense?